features/row_stripes.md

```diff
@@ -1,48 +1,44 @@
-# ** Title Here **
+# Row Stripes
 
-An example of adding stripes to a
-[`regular-table`](https://github.com/jpmorganchase/regular-table) by setting the
-each `tr`s `background-color`. Lets start by adding a `<regular-table>` to the
-page with an `id` that will be accessible on the window object using
-[`window.${id}`](https://stackoverflow.com/questions/18713272/why-do-dom-elements-exist-as-properties-on-the-window-object).
+Adding row stripes to a table, at its most basic, could be accomplished by
+adding `odd` and `even` css rules to each of the rows in the `<regular-table>`.
+Given this approach, a user would notice that on `"scroll"` the rows don't
+alternate, and the stripes remain fixed. The top-most row would always show the
+darker `background-color`, and the next row would retain its lighter, even
+`background-color` - repeating for each row and making the striping look
+inconsistent.
 
-```html
-<regular-table id="stripedRegularTable"></regular-table>
-```
-
-## Styling
+By alternating our css rules based on the oddness or evenness of the first row
+in the data set, we can ensure that the data and stripes style remain consistent
+as the user scrolls the table.
 
-For some basic stripes, we could simply add a `.stripes` class alternating the
-style on `tr:nth-child(odd)` and `tr:nth-child(even)`.
+# API
 
-```css
-.stripes tbody tr:nth-child(odd) td {
-    background-color: #eaedef;
-}
-
-.stripes tbody tr:nth-child(even) td {
-    background-color: white;
-}
+```html
+<regular-table id="example_table"></regular-table>
 ```
 
-However by simply adding stripes to the rows, the top-most row will always show
-with the darker `background-color: #ddd`, and the next row will retain its even,
-`background-color: #eee`, style as the user scrolls - repeating for each row and
-making the striping look inconsistent. We're going to add the `.reverse-stripes`
-class for use in our `alternateStripes()` function that applies a
-`StyleListener`.
+Lets set up by using `dataListener()` from `two_billion_rows` to create a data
+set and then call `setDataListener()` with it. Next we'll call our
+`alternateStripes()` function passing in the `<regular-table>` and then invoke
+`draw()` - checking that the `#stripedRegularTable` exists first. All of this
+will be invoked on `"load"`.
 
-```css
-.reverse-stripes tbody tr:nth-child(odd) td {
-    background-color: white;
-}
+```html
+<script type="module">
+    import { alternateStripes } from "./row_stripes.js";
+    import { dataListener } from "/dist/examples/two_billion_rows.js";
 
-.reverse-stripes tbody tr:nth-child(even) td {
-    background-color: #eaedef;
-}
+    window.addEventListener("load", () => {
+        const dl = dataListener(100, 50);
+        example_table.setDataListener(dl);
+        window.removeStripes = alternateStripes(example_table);
```

Comment:
This could be clearer - just remove `window.removeStripes`, or alternatively hook it up to like a `<button>` in the example perhaps?